raiden/tests/unit/transfer/mediated_transfer/test_initiatorstate.py

```diff
@@ -587,58 +583,6 @@ def test_refund_transfer_no_more_routes():
     assert sent_failed
 
 
-def test_refund_transfer_invalid_sender():
```

Comment:
The sender is now initialized from the balance proof, therefore it should never be invalid